patches/chrome-installer-mini_installer-mini_installer.cc.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/installer/mini_installer/mini_installer.cc b/chrome/installer/mini_installer/mini_installer.cc
-index 73e6240800f3c72bd96c91971c53a19516e1b0f1..e0049fab53dcb2bb8fe237523bf6eb9825c66c6e 100644
+index 73e6240800f3c72bd96c91971c53a19516e1b0f1..788f6dd2f3f90dc496688c7f0c412b490db23e77 100644
 --- a/chrome/installer/mini_installer/mini_installer.cc
 +++ b/chrome/installer/mini_installer/mini_installer.cc
-@@ -63,7 +63,7 @@ struct Context {
+@@ -27,6 +27,7 @@
+ // "Community Additions" comment on MSDN here:
+ // http://msdn.microsoft.com/en-us/library/windows/desktop/aa387694.aspx
+ #define SystemFunction036 NTAPI SystemFunction036
++#define StandardReferralCodeLen 6
```

Comment:
Maybe move this to the `#if defined(BRAVE_CHROMIUM_BUILD)` section below? It should use all caps and underscores to separate words also, to follow the Chromium standard.